### ISSUE TITLE
Infinite Loading Screen with AppImage After Updating to Ubuntu 22.04

### ISSUE THREAD

**Issue by notstarboard** [ytmdesktop/ytmdesktop#956]
### Consent

- [X] I verified that there is no open issue for the same subject.
- [X] I understand that **YTMDesktop have NO affiliation with Google or YouTube**

### Current Behavior

Ever since upgrading to Ubuntu 22.04 from 21.10, the YTM Desktop App gives an infinite loading screen. I've seen other bug reports about infinite loading screens until the window is resized, but resizing the window doesn't help here. 

I am able to open a guest window, sign into my Google account, and use the app normally. The main window will not load, though. I'm assuming there's some sort of sign-in issue preventing the page from loading, but I have no clue how to sign myself out so I can try to log in again. I tried removing all YTM data from the ~/.config folder, but that didn't help. I probably just need the browser equivalent of clearing my cookies.

### Expected Behavior

App loads as normal.

### Steps To Reproduce

1. Update from Ubuntu 21.10 to 22.04.
2. Launch YTM Desktop App.
3. Observe loading screen.
4. Sit for an eternity.
5. Continue to observe loading screen.

### YTMDesktop

1.13

### OS

Ubuntu

### OS Version

22.04

### Arch

x86

### Installation way

.AppImage

**Comment by Alipoodle** [comment]
Does opening from the command line or anything from the System logs display regarding any forms of Warnings/Errors which YTM may be giving?

**Comment by notstarboard** [comment]
Thanks for the reply! The command line spits out the following - not sure if it always does this or if it's related to the issue, though:

(node:2670) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
Update for version 1.13.0 is not available (latest version: 1.13.0, downgrade is disallowed).

I'll do some more digging into what that %appdata% equivalent might be - I'm honestly not sure! The information in ~/.config seemed to control things like window size and desktop app-specific settings, but clearly the login stuff lives somewhere else.

**Comment by notstarboard** [comment]
I unfortunately tried forcing a sign out via my Google account but that also didn't help.

Also I see there's a Session Storage folders within ~/.config, so maybe that _is_ where the login/session information lives. Maybe there's some other reason for the infinite loading wheel.

Would it be helpful for me to try out the Snap version? Snap is a deal-breaker for me long term but I'm happy to help gather some data!

**Comment by notstarboard** [comment]
Sorry to spam, but one last data point - I just confirmed this issue is also present on another computer that went through the same upgrade path.

**Comment by giuricich** [comment]
hey i think i have the same issue.
im trying to npm start the app from source but all i get is the infinite loading screen and terminal spits out this:
```
node:46693) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. 
Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
```
when i stop execution this comes out
```
^C/home/potato/dev/ytmdesktop/node_modules/electron/dist/electron exited with signal SIGINT
npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! youtube-music-desktop-app@1.13.0 start: `electron .`
npm ERR! Exit status 1
npm ERR! 
npm ERR! Failed at the youtube-music-desktop-app@1.13.0 start script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /home/potato/.npm/_logs/2022-05-02T17_07_54_179Z-debug.log
```
[here is a pastebin of the log for reference](https://pastebin.com/Ay5uxua6)
not 100% sure if its the same issue, but it seems really similar

OS: Fedora 35 x64
node: v12.22.12
npm: v6.14.16

**Comment by Petemir** [comment]
This is the error I am getting trying to run from the command line.

```
Checking for update
(node:398938) UnhandledPromiseRejectionWarning: Error: ERR_FAILED (-2) loading 'https://music.youtube.com'
    at rejectAndCleanup (electron/js2c/browser_init.js:205:1493)
    at Object.stopLoadingListener (electron/js2c/browser_init.js:205:1868)
    at Object.emit (events.js:327:22)
(node:398938) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:398938) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
Update for version 1.13.0 is not available (latest version: 1.13.0, downgrade is disallowed).
```

Kubuntu, 22.04, running from AppImage.

**Comment by wez** [comment]
Same error on Fedora 35; this is my first time trying to run this app:

```
; ./YouTube-Music-Desktop-App-1.13.0.AppImage
Checking for update
(node:77634) [DEP0005] DeprecationWarning: Buffer() is deprecated due to security and usability issues. Please use the Buffer.alloc(), Buffer.allocUnsafe(), or Buffer.from() methods instead.
Generated new staging user ID: c23a628e-225e-5a54-9941-9348b878b37d
Update for version 1.13.0 is not available (latest version: 1.13.0, downgrade is disallowed).
```

**Comment by Alipoodle** [comment]
Duplicate of #916
Please can you continue any convos in there!
